pom.xml

```diff
@@ -56,6 +56,7 @@
 		<prometheus.version>0.6.0</prometheus.version>
 		<cloudfoundry.client.version>3.21.0.RELEASE</cloudfoundry.client.version>
 		<logback.contrib.version>0.1.5</logback.contrib.version>
+		<kotlin.version>1.3.61</kotlin.version>
```

Comment:
If we add kotlin, we also should add support for static code checks (in SonarCube) for it.

Reply:
decided to be postponed. Put a reminder at https://github.com/promregator/promregator/projects/2#card-34092678